app/helpers/users_helper.rb

```diff
@@ -4,4 +4,8 @@ module UsersHelper
   def current_user_name(user)
     [user.name, user.email].find(&:present?)
   end
+
+  def user_name(user)
+    user&.name.presence || user&.email.presence || t('views.common.withdrawn')
```

Comment:
`&.`は一種のif文なので、脳の負担を減らすために必要以上に増やしたくないなー、と思う人です。

```suggestion
    user ? user.name.presence || user.email : t('views.common.withdrawn')
```